TombEngine/Scripting/Internal/TEN/Input/InputHandler.cpp

```diff
@@ -37,61 +37,82 @@ namespace TEN::Scripting::Input
 		return true;
 	}
 
-	/// Check if an action key is being held.
-	// @function KeyIsHeld
-	// @tparam Input.ActionID action Action ID to check.
-	static bool KeyIsHeld(int actionID)
+	/// Check if an action key is being hit.
+	// @function IsKeyHit
+	// @tparam Input.ActionID actionID Action ID to check.
+	static bool IsKeyHit(int actionID)
 	{
-		if (!CheckInput(actionID))
+		if (!IsValidAction(actionID))
 			return false;
 
-		if (IsHeld((InputActionID)actionID))
-			return true;
+		return IsClicked((ActionID)actionID);
+	}
+
+	/// Check if an action key is being held.
+	// @function IsKeyHeld
+	// @tparam float[opt=0] delaySec Delay time in seconds before a hold can be registered.
```

Comment:
Please change order of `tparam` here to correspond to actual function signature.